stand_with_ukraine.js

```diff
@@ -1,7 +1,7 @@
+/* global swu_options */
 document.addEventListener('DOMContentLoaded', function() {
-	let body = document.getElementsByTagName('body')[0];
-	let div  = document.createElement('div');
+	const div  = document.createElement('div');
```

Comment:
const and let are supported after IE 11 - maybe is worth to support older browsers?
https://caniuse.com/const

Reply:
Switching to `const` doesn't help support older browsers. `var` would be needed for that. Probably worth making that change.
```suggestion
	let div  = document.createElement('div');
```

Reply:
yes exactly, To be compatible with IE the first step was exactly that! 😅